src/components/tabs/tabs.component.tsx

```diff
@@ -40,12 +79,19 @@ const Tabs = ({
   headerWidth,
   showValidationsSummary,
   ...rest
-}) => {
+}: TabsProps) => {
+  if (position !== "left" && headerWidth !== undefined) {
+    // eslint-disable-next-line no-console
+    console.error(
```

Comment:
Should this be an `invariant` rather than a console error?

Reply:
Well that's open to debate. My first attempt was to make it an `invariant` but that means the test for this fails, as it simply crashes the component with an error. Apparently `console.error` is called by errors in custom PropTypes which is what we had before so this was the closest equivalent. Consumers have in the past complained at us completely crashing their app when they've done something that's only mildly questionable - so I'm not sure we want to introduce a new reason for that.

Reply:
I had a little think about this more (I agree with @robinzigmond) and perhaps a package like [this](https://www.npmjs.com/package/warning) would be better for us to use in instances like this. Not suggesting it as part of this PR as I'd prefer to do it across all components at once if we agree it's a good idea